students/charlie_england/session05/mailroom_part3.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+#refactor to have the donors dict be 2 lists
+import sys
+# Donors dict Name:[[list of donations],[sum donations, num donations, average donations]]
+donors = {"Chris Christly": [[1000.21, 250.80],[]], "Bob Barley": [[800.33],[]], "Nick Nilly": [[
```

Comment:
You only want the values to be 1 list, it looks like you have a list of lists.

Ex:
donors = {"Chris Christly": [1000.21, 250.80], "Bob Barley": [800.33], ... } 

This makes it easier below to not have to do the 0 index